src/index.js

```diff
@@ -0,0 +1,465 @@
+
+/* eslint-disable */
+
+import domListeners from './modules/domListeners';
+
+import deleteTask from './todo/deleteTask';
+import viewTask from './todo/viewTask';
+import completedTask from './todo/completedTask';
+import storage from './storage';
+import removeProject from './project/removeProject';
+import filterTask from './project/filterTask';
+import editTask from './todo/edit';
+import editProject from './project/editProject';
+
+const format = require('date-fns/format');
```

Comment:
##  Fix the import errors with most of the files
- [ ] We have an issue importing these files on our `index.js` module. Do you mind fixing this?
```suggestion
import domListeners from './modules/domListeners';

import deleteTask from './modules/todo/deleteTask.js';
import viewTask from './modules/todo/viewTask.js';
import completedTask from './modules/todo/completedTask.js';
import storage from './modules/storage.js';
import removeProject from './modules/project/removeProject';
import filterTask from './modules/project/filterTask.js';
import editTask from './modules/todo/edit.js';
import editProject from './modules/project/editProject.js';

const format = require('date-fns/format');```